### ISSUE TITLE
Switch to custom login pages and add rate limiting

### ISSUE THREAD

**Issue by mlissner** [freelawproject/bigcases2#54]
I know this feels a bit silly, because we don't yet have users, but I regret to share that the django admin page doesn't have rate limiting, making it vulnerable to brute force attacks.

There are two ways to fix this:
1. Tweak the admin page to make it secure (doable, I think?)
2. Redirect the admin page to the regular login page (this is what we do in CL)
3. Disable the admin login somehow?

I think option 2 is the best one, but it means making a login page and a forgot password system. 

We'll need those eventually anyway, I suppose.
 
The normal tool for this is [django-ratelimit](https://github.com/jsocol/django-ratelimit/), but we should remember that in https://github.com/freelawproject/courtlistener/issues/2133, we had to add some custom header processing to make it work properly in k8s.

**Comment by mlissner** [comment]
I think we can launch without this, but I know that the security kids are going to be all over it asking for bounties if we don't have it.

**Comment by mlissner** [comment]
In fact, our VDP says that bots.law would be out of scope by default, so I think once this is complete, we should add bots.law to the VDP: https://free.law/vulnerability-disclosure-policy/

**Comment by mlissner** [comment]
This is done enough for now. When we need the rest of the login features, we'll open a fresh issue for that.
